### ISSUE TITLE
Create API for Marketplace

### ISSUE THREAD

**Issue by shivaalipour** [GSA/marketplace-fedramp-gov#56]
Feedback from user on creating an API: 

Hi Team

I love the improvements to FedRAMP marketplace and have already done a deep dive analysis.  My problem is that it’s a snapshot in time and it’s hard to track “updates.”  Do you have an API I can use to keep data accurate and up to date?

Thanks!
--

Josh Dries

Partner Development Manager

Google Cloud Public Sector

901.210.3010

jdries@google.com

**Comment by danstoll** [comment]
Yes, please.. being able to monitor by package ID would be sweet.

**Comment by Chris-Lauer** [comment]
The data json is api-like, I've been playing around with downloading it and parsing for changes
